bin/console-conf-wrapper

```diff
@@ -79,4 +79,8 @@ fi
 
 cat /usr/share/subiquity/console-conf-wait
 read REPLY
+# start the console-conf routine in snapd to delay any new refreshes, and wait
+# for current refreshes to complete, this will print off messages if there are
+# on-going refreshes
+snap routine console-conf-start
```

Comment:
```suggestion
if snap routine console-conf-start --help >/dev/null 2>/dev/null; then
    snap routine console-conf-start
fi
```

Reply:
@anonymouse64 does above make sense? we test for help, which works with any snap, and then will call it if it is available.

Reply:
yes looks good to me, thanks for the suggestion I will commit it here